recipes/react-router/app/routes/puck-splat.tsx

```diff
@@ -0,0 +1,93 @@
+import { useFetcher, useLoaderData } from "react-router";
+import type { Data } from "@measured/puck";
+import { Puck, Render } from "@measured/puck";
+
+import "@measured/puck/puck.css";
+
+import type { Route } from "./+types/puck-splat";
```

Comment:
What is this `+` import magic?!

Reply:
This is something new in RR7. Types are auto generated based on file changes for the route. Works really well.